projects/skia/image_filter_deserialize.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+max_len = 10024
+timeout = 10
```

Comment:
Yeah, I can't really imagine a case in which something that takes 10 seconds or more is good for fuzzing - I'd rather know about it and be able to write a guard to prevent that.